src/overlays/actors/ovl_En_Zf/z_en_zf.c

```diff
@@ -452,9 +452,15 @@ s16 EnZf_FindNextPlatformAwayFromPlayer(Vec3f* pos, s16 curPlatform, s16 arg2, G
         }
     }
 
-    // These functions have no side effects, so these two calls do nothing
+    //! @bug `altNextPlatform` can be -1 in certain conditions and cause an out of bounds access.
+    //! Under normal conditions, this doesn't cause problems because the data before `sPlatformPositions`
+    //! is section padding between .text and .data, so 0 gets read as a float.
+    //! Other compilers can shift things and result in a crash. Defining `AVOID_UB` will remove these function calls
+    //! entirely as they are dead code and do not have any effect.
```

Comment:
```suggestion
    //! entirely as they have no side effect.
```
it's not dead code (it gets run), and the proper computer science term is indeed "side effect" like in the current comment